shared/origami/python/setup.py

```diff
@@ -15,7 +15,7 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(description="Origami Python Bindings Setup Script")
-    parser.add_argument("--source", "-s", type=str, default=Path(__file__).parent.parent.parent.resolve() / "Source", help="Path to TensileLite source directory.")
+    parser.add_argument("--source", "-s", type=str, default=Path(__file__).parent.parent.parent.resolve() / "origami" / "src", help="Path to origami source directory.")
```

Comment:
This can be simplified.
```suggestion
    parser.add_argument("--source", "-s", type=str, default=Path(__file__).parent.parent.resolve() / "src", help="Path to origami source directory.")
```